collector/overhead_alert_metrics.go

```diff
@@ -128,9 +179,10 @@ func (r *ReconcileOverhead) Reconcile(ctx context.Context, request reconcile.Req
 				log.V(4).Info(fmt.Sprintf("filtering scheduling metric for %s with gap %v and total %v",
 					request.NamespacedName.String(), scheduleDuration, totalDuration))
 			}
-		} else {
-			return reconcile.Result{}, tagPipelineRunsWithTaskRunsGettingThrottled(pr, r.client, ctx)
 		}
+	} else {
```

Comment:
this is the path where we are still running ... aka the succeed condition is unknown